com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -852,6 +853,28 @@ private void OnEnable()
             }
         }
 
+        private void Awake()
+        {
+            Application.wantsToQuit += ApplicationWantsToQuit;
```

Comment:
what about this instead?
https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnApplicationQuit.html

Reply:
Yeah I think we should use OnApplicationQuit instead.